### PR TITLE
fix!: remove positional arguments for artifact commands

### PR DESCRIPTION
**Pull request by wtrocki**
1. Positionals for artifact id/name were removed 
2. Positional for file input for create command is the only positional we have
3. For file output we use flag as suggested by @craicoverflow 

## Verification

```
## Create artifact in my-group from schema.json file
rhoas service-registry artifact create --artifact-id=my-artifact --group=my-group artifact.json

## Get artifact content
rhoas service-registry artifact get --artifact-id=my-artifact --group=my-group --file-location=artifact.json

## Get artifact content by hash
rhoas service-registry artifact download --hash=cab4...al9 --file-location=artifact.json

## Delete artifact
rhoas service-registry artifact delete --artifact-id=my-artifact

## Get artifact metadata
rhoas service-registry artifact metadata --artifact-id=my-artifact --group=my-group

## Update artifact
rhoas service-registry artifact update --artifact-id=my-artifact artifact-new.json

## List Artifacts
rhoas service-registry artifact list --group=my-group --limit=10 page=1

## View artifact versions
rhoas service-registry artifact versions --artifact-id=my-artifact --group=my-group
```

### CODE REVIEW

**Comment by wtrocki** [comment]
@rkpattnaik780 Ready for review

File: pkg/cmd/registry/artifact/crud/update/update.go [hunk 1]
```diff
@@ -57,17 +57,19 @@ func NewUpdateCommand(f *factory.Factory) *cobra.Command {
...
 Artifacts can be typically in JSON format for most of the supported types, but may be in another format for a few (for example, PROTOBUF).
-The type of the content should be compatible with the artifact's type.
-(it would be an error to update an AVRO artifact with new OPENAPI content, for example).
+The type of the content should be compatible with the current artifact's type.
 
 When successful, this creates a new version of the artifact, making it the most recent (and therefore official) version of the artifact.
 
 An artifact is update using the content provided in the body of the request.  
-This content is updated under a unique artifactId provided by user.
+This content is updated under afu unique artifactId provided by user.
```
Comment (rkpattnaik780) [inline pkg/cmd/registry/artifact/crud/update/update.go:65]:
```suggestion
This content is updated under a unique artifactId provided by user.
```

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]
Added few minor suggestions.
 Few things I noticed that aren't related to this PR:

This error message can be formatted a bit
Screenshot from 2021-08-20 16-09-59

Should we add few punctuation - `Do you want to delete artifact "<name>" from group "default"` ?
Screenshot from 2021-08-20 16-13-12

File: pkg/cmd/registry/artifact/crud/update/update.go (hunk 1 above)
Comment (wtrocki) [reply pkg/cmd/registry/artifact/crud/update/update.go:65]: ROFL :D

**Review by rkpattnaik780 — APPROVED** [review]

**Pull request merged**